ports/dotenv/portfile.cmake

```
@@ -0,0 +1,16 @@
+vcpkg_from_github(
```

```suggestion
set(VCPKG_BUILD_TYPE release) # header-only library
vcpkg_from_github(
```

Recommended that header only libraries only build release flavor.